docs/core-concepts/stateconfig.md

```diff
@@ -22,6 +23,20 @@ state:
 
 The `stateFile` under the `options` section is mandatory for local backend.
 
+## Kubernetes Backend
+
+```yaml
+varFile: variables-gitops.yaml
+
+toolFile: tools-gitops.yaml
+
+state:
+  backend: k8s
+  options:
+    namespace: devstream        # optional, default is "devstream", will be created if not exists
+    configmap: devstream-state  # optional, default is "devstream-state", will be created if not exists
```

Comment:
How about to use ConfigMap name "state"? devstream/state vs devstream/devstream-state